torch/nested/_internal/nested_tensor.py

```diff
@@ -420,8 +420,8 @@ def _nt_view_dummy() -> torch.Tensor:
     global _dummy_instance
     if _dummy_instance is None:
         _dummy_instance = NestedTensor(
-            values=torch.randn(3, 3, device="meta"),
-            offsets=torch.randint(3, (2,), device="meta", dtype=torch.int64),
+            values=torch.zeros(3, 3, device="meta"),
+            offsets=torch.zeros(3, device="meta", dtype=torch.int64),
```

Comment:
Is it possible to add a test case for what vincent was running into?

Reply:
I tried but wasn't able to repro it after #122704 landed. I decided to make the change here anyway to avoid any other unforeseen randomness-related problems